samples/bootloader/CMakeLists.txt

```diff
@@ -9,4 +9,22 @@ cmake_minimum_required(VERSION 3.8.2)
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(bootloader)
 
-target_sources(app PRIVATE src/main.c)
+# This allows the bootloader sample to be built as a separate image in
```

Comment:
s/the bootloader sample/this secure boot image/

As this sample is written, it's not obvious enough whether we are looking at a sample that uses a bootloader, or a sample that is a bootloader.